packages/react/src/products/hooks/__tests__/useProductListing.test.tsx

```diff
@@ -249,6 +253,47 @@ describe('useProductListing', () => {
         },
       });
     });
+
+    it('should return data correctly when `isACustomListingPage` is true', () => {
+      const slug = getSlug('en/customlistingpage', true);
+
+      const { result } = renderHook(
+        () => useProductListing(slug, { isACustomListingPage: true }),
+        {
+          wrapper: withStore(mockCustomListingPageState),
+        },
+      );
+
+      const mockList =
+        mockProductsListNormalizedPayload.entities.productsLists[
+          mockCustomListingPageHash
+        ];
+
+      expect(fetchCustomListing).not.toHaveBeenCalled();
```

Comment:
I would not add this expect here and would add another test that would test if when enableAutoFetch is true, there is no data in the store and isCustomListingPage is true, that it would call fetchCustomListing.